packages/botonic-api/src/data-provider/dynamodb-data-provider.ts

```diff
@@ -56,7 +57,14 @@ export class DynamoDBDataProvider implements DataProvider {
   }
 
   // @ts-ignore
-  async getUsers(limit = 10, offset = 0): Promise<User[]> {} // TODO: Implement
+  async getUsers(limit = 10, offset = 0): Promise<User[]> {
+    // TODO: offset really needed here?
+    const result = await this.userEventsTable.scan({
+      filters: { attr: SORT_KEY_NAME, beginsWith: USER_PREFIX },
+      limit,
+    })
+    return result.Items
+  }
```

Comment:
If `getUsers` has been called from an api request with pagination, `offset` is needed in order to return only the specific users from the desired page.

With this implementation you're only returning always the first users fixed by the `limit` value.
We should allow pagination here too using the `offset` argument.

Reply:
we should check how this can be done with dynamoDB, leaving the TODO here as this should be reviewed but out of the scope of this PR.